src/libsystemd/sd-netlink/netlink-types.c

```diff
@@ -24,10 +24,11 @@
 #include <linux/can/netlink.h>
 #include <linux/in6.h>
 #include <linux/veth.h>
-#include <linux/if_bridge.h>
-#include <linux/if_addr.h>
 #include <linux/if.h>
 #include <linux/ip.h>
+#include <linux/if_addr.h>
+#include <linux/if_addrlabel.h>
+#include <linux/if_bridge.h>
```

Comment:
why this reorder? We usually try to keep includes in alphabetical order, unless the kernel headers are borked and don't permit that